tests/suites/test_suite_base64.data

```diff
@@ -1,33 +1,3 @@
-mask_of_range empty (1..0)
```

Comment:
Let's not move the base64 tests since we know we actually want them right where they are.